### PR TITLE
🐛 Avoid displaying confusing message when editing workbench

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?

- Fixes ITISFoundation/osparc-issues#1189

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js [hunk 1]
```diff
@@ -634,7 +634,8 @@ qx.Class.define("osparc.desktop.StudyEditor", {
         })
         .catch(error => {
           if ("status" in error && error.status === 409) {
-            osparc.FlashMessenger.getInstance().logAs(error.message, "ERROR");
+            console.log("Flash message blocked") // Workaround for osparc-issues #1189
+            // osparc.FlashMessenger.getInstance().logAs(error.message, "ERROR");
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js:638]: please do not leave commented code

**Review by pcrespov — CHANGES_REQUESTED** [review]
TIP to make a good PR:
- create a readable title since it will display in the release notes
- move the issue number as an enumeration to the message. It renders and links the issues
- add the keyword "close" or "fixes" to automatically closed the resolved case

SEE https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js (hunk 1 above)
Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/desktop/StudyEditor.js:638]: Haha was just writing the same

**Review by jsaq007 — CHANGES_REQUESTED** [review]

**Review by pcrespov — APPROVED** [review]

**Review by elisabettai — APPROVED** [review]
That was quick, thank you!

**Review by sanderegg — APPROVED** [review]
and thanks a lot!

**Review by GitHK — APPROVED** [review]

**Review by jsaq007 — APPROVED** [review]

**Pull request merged**